server/service/src/sync/program_schemas/hiv_testing_program.json

```diff
@@ -44,6 +51,28 @@
       "required": ["enrolmentDatetime", "status"],
       "type": "object"
     },
+    "Note": {
+      "properties": {
+        "authorId": {
+          "description": "authorId should be captured automatically from app state",
```

Comment:
have these come from somewhere? some descriptions sound like developer comments 😉

Reply:
They're auto-generated by the programschemas repo based on comments. Won't show up in the UI.

Reply:
We experimented with showing them as help text in the UI though... But in this case there is a special component that renders the note anyway.